src/test/java/ru/training/at/hw3/Ex2.java

```diff
@@ -0,0 +1,162 @@
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Ex2 extends DriverManager {
+
+    DriverManager driverManager = new DriverManager();
+    private final LoginForm loginFormPage;
+    private final HeaderOfPage headerOfPage;
+    private final SidebarMenu sidebarMenu;
+    private final ElementsPage elementsPage;
+
+    public Ex2() {
+        driverManager.setupDriver();
+        PageFactory.initElements(driver, this);
+        loginFormPage = new LoginForm(driver);
+        headerOfPage = new HeaderOfPage(driver);
+        sidebarMenu = new SidebarMenu(driver);
+        elementsPage = new ElementsPage(driver);
+
+    }
+
+    public ElementsPage getElementsPage() {
+        return elementsPage;
+    }
+
+    public LoginForm getLoginFormPage() {
+        return loginFormPage;
+    }
+
+    public HeaderOfPage getHeaderOfPage() {
+        return headerOfPage;
+    }
+
+    public SidebarMenu getSidebarMenu() {
+        return sidebarMenu;
+    }
+
+    @FindBy(css = "ul.uui-navigation.navbar-nav.navbar-right")
+    private WebElement loginForm;
+    @FindBy(css = "#login-button")
+    private WebElement loginButton;
+    @FindBy(id = "name")
+    private WebElement nameField;
+    @FindBy(id = "password")
+    private WebElement passField;
+    @FindBy(id = "user-name")
+    private WebElement userName;
+    @FindBy(css = "body > header > div > nav > ul.uui-navigation.navbar-nav.navbar-right"
+            + " > li > div > div > button")
+    private WebElement loggedButton;
+    @FindBy(css = "a.dropdown-toggle")
+    private WebElement openMenuService;
+    @FindBy(linkText = "DIFFERENT ELEMENTS")
+    private WebElement openDifferentElements;
+
+
+    public void openTestSite() {
+        driver.get(Locators.getHomepage());
+        Assert.assertEquals(driver.getCurrentUrl(),
+                Locators.getHomepage());
+    }
+
+    public void browserTitleAssertion() {
+        Assert.assertEquals(driver.getTitle(), Locators.getTitle());
+    }
+
+    public void login() {
+        loginFormPage.getLoginForm().click();
+        loginFormPage.getNameField().sendKeys(Locators.getName());
+        loginFormPage.getPassField().sendKeys(Locators.getPassword());
+        loginFormPage.getLoginButton().click();
+        Assert.assertEquals(loginFormPage.getLoggedButton().getText(),
+                Locators.getMessageForLoginCheck());
+    }
+
+    public void loginAssertion() {
+        Assert.assertEquals(loginFormPage.getUserName().getText(),
+                Locators.getNameForLoginCheck());
+    }
+
+    public void openDifferentElementsPage() {
+        openMenuService.click();
+        openDifferentElements.click();
+    }
+
+    public void selectCheckboxes() {
+        for (WebElement checkbox : elementsPage.getCheckboxes()) {
+            if (checkbox.getText().equals(Locators.getWater()) | checkbox
+                    .getText().equals(Locators.getWind())) {
```

Comment:
Please wrap lines in move obvious way
```suggestion
            if (checkbox.getText().equals(Locators.getWater()) 
                || checkbox.getText().equals(Locators.getWind())) {
```